PD-213 Update 22.12.2 Release Notes (#1861)
truenas/documentation: content/SCALE/SCALE22.12.md

### Component Versions

TrueNAS SCALE is built from many different software components.

> This list has up to date information on which versions of Linux, ZFS, and NVIDIA drivers included with this TrueNAS SCALE release.

Reviewer: ```suggestion
This list has up-to-date information on which versions of Linux, ZFS, and NVIDIA drivers are included with this TrueNAS SCALE release.
```

Author: fixed in my local copy to push in next commit.

| 22.12.2 | <a href="https://ixsystems.atlassian.net/browse/NAS-121035" target="_blank">NAS-121035</a> | Web UI not updating pool column in Storage > Disks page (disk.get_unused) | If a disk, associated with an exported zpool, and where you perform a Disk Wipe operation, the pool table does not update and the disk shows the wrong state. | 22.12.3<br> 23.10-ALPHA.1 Cobia |
| 22.12.2 | <a href="https://ixsystems.atlassian.net/browse/NAS-121030" target="_blank">NAS-121030</a> | ES12 Enclosure View not updating after drive insertion and recognition (HA/SCALE) | An HA system Enclosure View for an ES12 does not update after drive insertion and recognition by SCALE. | 22.12.3 |
| 22.12.2 | <a href="https://ixsystems.atlassian.net/browse/NAS-121014" target="_blank">NAS-121014</a> | System configured on LA time but Network dashboard card is in EST | System time configured PST while the main Dashboard Network card displays EST. | 22.12.3<br> 23.10-ALPHA.1 Cobia |

> | 22.12.2 | <a href="https://ixsystems.atlassian.net/browse/NAS-120961" target="_blank">NAS-120961</a> | HA: Connected TrueCommand Cloud status is disabled after HA failover | Add an HA system to TrueCommand, failover of the HA system results in TrueCommand cloud status change to Disabled after the failover. TrueCommand should remain enabled and connected. | 22.12.3 |

Reviewer: ```suggestion
| 22.12.2 | <a href="https://ixsystems.atlassian.net/browse/NAS-120961" target="_blank">NAS-120961</a> | HA: Connected TrueCommand Cloud status is disabled after HA failover | If you add an HA system to TrueCommand and perform a failover, TrueCommand cloud disables. TrueCommand should remain enabled and connected. | 22.12.3 |
```
